ocw-ui/frontend/app/index.html

```diff
@@ -143,5 +143,21 @@ <h5 id="OCW-powered">Powered by Apache Open Climate Workbench</h5>
 
     <!-- grunt wiredep wasn't properly linking this. Had to add manually -->
     <script src="bower_components/angular-ui-date/src/date.js"></script>
+
+    <!-- Footer for the Web-App -->
+    <footer class="footer" align="center">
+      <div class="container">
+        <hr>
+        <p>
+          Apache Open Climate Workbench Software . 
+          <a href="https://climate.apache.org/" target="_blank">About Us</a> . <a href="https://issues.apache.org/jira/browse/CLIMATE/?selectedTab=com.atlassian.jira.jira-projects-plugin:issues-panel" target="_blank">Report an issue</a> .
```

Comment:
The URL here can be abbreviated to https://issues.apache.org/jira/browse/CLIMATE or a mailto link which points to dev@climate.apache.org. Non technical people may be more likely to use the mailto link.